docs/basics.txt

```diff
@@ -102,13 +92,11 @@ The JanusGraph distribution contains a command line Console which makes it easy
 graph = JanusGraphFactory.open('path/to/configuration.properties')
 ----
 
-JanusGraph Embedded
-+++++++++++++++++++
+===== JanusGraph Embedded
```

Comment:
We already differentiate between LOCAL and REMOTE deployments. And refer to BerkeleyJE as an embedded persistence engine. Somewhere I also think I saw "embedded Cassandra" in Janus. Shall we call this "serverless" or "headless" mode to avoid confusion with other uses of the word embedded?